charts/promtail/values.yaml

```
@@ -481,6 +506,9 @@ config:
     limits_config:
       {{- tpl .Values.config.snippets.extraLimitsConfig . | nindent 2 }}
 
+    tracing:
+      {{- tpl .Values.config.snippets.tracingConfigs . | nindent 2 }}
```

I'd set `config.enable_tracing: false` above, then change this to:

```suggestion
      enabled: {{ .Values.config.enable_tracing }}
```

The Jaeger env vars should be set by the user with `extraEnv` (this is how the other Grafana stack charts are configured).